opensrp-core/src/test/java/org/smartregister/sync/intent/SyncIntentServiceTest.java

```diff
@@ -449,6 +454,60 @@ public void testPushECToServer() throws Exception {
 
     }
 
+    @Test
+    public void testPushECToServerVerifyMarkEventsAsSyncedForFailedEventsAndClients() throws Exception {
+        syncIntentService = spy(syncIntentService);
+        List<JSONObject> clients = new ArrayList<>();
+
+        JSONObject client = new JSONObject(clientJson);
+        clients.add(client);
+
+        List<JSONObject> events = new ArrayList<>();
+        JSONObject event = new JSONObject(eventJson);
+        events.add(event);
+
+        Map<String, Object> pendingEvents = new HashMap<>();
+        pendingEvents.put(AllConstants.KEY.CLIENTS, clients);
+        pendingEvents.put(AllConstants.KEY.EVENTS, events);
+
+        JSONObject expectedRequest = new JSONObject();
+        Object value = pendingEvents.get(AllConstants.KEY.CLIENTS);
```

Comment:
I do not see anywhere else where this `value` (clients) object is being used apart from the next line. Can we just use pendingEvents#get inside expectedResult#put the way it is done for events?